ci/clang-tidy-all-new.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
```

Comment:
This script for easy local checking. Except Common++ all directories are ignored. It will be replacing the original script after all folders are fixed